pkg/config/config_template.yaml

```diff
@@ -1601,6 +1601,14 @@ api_key:
 #
 # dogstatsd_origin_detection: false
 
+## @param dogstatsd_origin_detection_client - boolean - optional - default: false
+## @env DD_DOGSTATSD_ORIGIN_DETECTION_CLIENT - boolean - optional - default: false
+## A DogStatsD client can send its container ID.
+## Enable this parameter to instruct the Agent to use the container ID to enrich the metrics with container tags.
```

Comment:
Would this make sense?
```suggestion
## Whether the Agent should use a client-provided container ID to enrich the metrics, events and service checks with container tags.
```

Would it make sense to mention that this affects dogstatsd_tag_cardinality above `low` here?

Reply:
> Would it make sense to mention that this affects dogstatsd_tag_cardinality above low here?

I'm not sure I understand, mind clarifying ?

Reply:
Nevermind, I got confused and thought `low` cardinality tags are somehow not derived from the origin.